File: hrms/src/main/java/com/konstantinlevin77/hrms/business/abstracts/JobseekerService.java
package com.konstantinlevin77.hrms.business.abstracts;

import java.util.List;

import com.konstantinlevin77.hrms.adapters.NationalIdVerifierAdapter.NationalIdVerifier;
import com.konstantinlevin77.hrms.core.results.abstracts.DataResult;
import com.konstantinlevin77.hrms.core.results.abstracts.Result;
import com.konstantinlevin77.hrms.dataAccess.abstracts.JobseekerDao;
import com.konstantinlevin77.hrms.entities.concretes.Jobseeker;
import com.konstantinlevin77.hrms.entities.dtos.JobseekerWithoutCvDto;

public interface JobseekerService {
	
	public DataResult<List<Jobseeker>> getAll();
	
	public Result add(JobseekerWithoutCvDto jobseeker,NationalIdVerifier nationalIdVerifier);

	public Result verify(int id);
	
}

File: hrms-frontend/src/services/JobseekerService.js
import axios from 'axios'

export default class JobseekerService {

    getAll(){
        return axios.get("localhost:8080/api/jobseekers/getall")
    }

    add(jobseeker){
        return axios.post("localhost:8080/api/jobseekers/add",jobseeker)
    }

    verify(id){
        return axios.post("localhost:8080/api/jobseekers/verify",id)
    }

}